### DISCUSSION TITLE
Can't install python 3.10.3

### DISCUSSION THREAD

**Discussion by N-i-l-l** [pyenv/pyenv#2993] [Q&A] [answered]
The terminal gives me this:
python-build: use openssl from homebrew
python-build: use readline from homebrew
Installing Python-3.10.3...
python-build: use readline from homebrew
python-build: use ncurses from homebrew
python-build: use zlib from xcode sdk

And then just doesn't do anything, and I have to cancel it or it keeps running. Any help? I'm on macOS Monterey

**Comment by native-api** [comment, marked as answer]
That's expected, the compilation takes some time. You can add the `-v` option to see its progress.

If it does hang, please create an issue and add diagnostic information as per the issue template.
